modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122a.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
```

Comment:
Please organize your code so that methods appear as they're called (e.g. `#template_options` should appear before `#page1_options`, which should appear before `#page1_options`, etc.

Reply:
Done!